### PR TITLE
Enhance EDA tables using framework features

### PR DESCRIPTION
**Pull request by jamestalton**
Added expanded columns and description columns.
Added EDA Cells to display projects, rulebooks, etc in the tables. 
Enhanced card views to show counts section.

Screenshot 2023-04-14 at 4 00 44 PM

### CODE REVIEW

File: cypress/e2e/eda/Decision-Environments/decision-env-crud.cy.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ describe('EDA decision environment- Create, Edit, Delete', () => {
   it('can delete a decision environment from the details page', () => {
     cy.createEdaDecisionEnvironment().then((edaDE) => {
       cy.navigateTo(/^Decision environments$/);
-      cy.get('h1').should('contain', 'DecisionEnvironments');
+      cy.hasTitle(/^Decision environments$/);
       cy.get('button[aria-label="table view"]').click();
```
Comment (akus062381) [inline cypress/e2e/eda/Decision-Environments/decision-env-crud.cy.ts:61]: This change removes an assertion. if you want to make this change, let's update the custom command to include a specific assertion

File: cypress/e2e/eda/Decision-Environments/decision-env-crud.cy.ts [hunk 2]
```diff
@@ -69,8 +69,8 @@ describe('EDA decision environment- Create, Edit, Delete', () => {
       cy.clickModalButton('Delete decision environments');
       cy.wait('@deleteDE').then((deleteDE) => {
         expect(deleteDE?.response?.statusCode).to.eql(204);
-        cy.hasTitle(/^DecisionEnvironments$/);
       });
+      cy.hasTitle(/^Decision environments$/);
     });
```
Comment (akus062381) [inline cypress/e2e/eda/Decision-Environments/decision-env-crud.cy.ts:74]: This change removes an assertion. if you want to make this change, let's update the custom command to include a specific assertion

**Review by akus062381 — CHANGES_REQUESTED** [review]

**Review by akus062381 — APPROVED** [review]

File: cypress/e2e/eda/Decision-Environments/decision-env-crud.cy.ts (hunk 1 above)
Comment (jamestalton) [reply cypress/e2e/eda/Decision-Environments/decision-env-crud.cy.ts:61]: updated

File: cypress/e2e/eda/Decision-Environments/decision-env-crud.cy.ts (hunk 2 above)
Comment (jamestalton) [reply cypress/e2e/eda/Decision-Environments/decision-env-crud.cy.ts:74]: updated

**Pull request merged**